simple_calculator/myscript.js

```diff
@@ -2,10 +2,16 @@ const showNumber = document.getElementById("result")
 let resultNumber = Number(showNumber.innerText)
 
 document.addEventListener("click", function(e) {
-    console.log("clicked")
+    
     if (e.target.className.includes("cal-button")) {
+        console.log("cal-button clicked")
         resultNumber = resultNumber + Number(e.target.innerText)
         showNumber.innerText = resultNumber
         console.log(showNumber.innerText)
     }
+
+    if (e.target.className.includes("reset")) {
+        console.log("reset clicked")
+        resultNumber = Number(0);
```

Comment:
0은 이미 숫자형이니 명시적 형변환을 해주지 않아도 될 것 같네요.

```suggestion
        resultNumber = 0;
```

[참고] 암시적 형변환과 명시적 형변환
https://medium.com/gdana/%EC%9E%90%EB%B0%94%EC%8A%A4%ED%81%AC%EB%A6%BD%ED%8A%B8%EC%9D%98-%ED%98%95%EB%B3%80%ED%99%98%EC%9D%80-%EB%91%90%EA%B0%80%EC%A7%80%EB%8B%A4-b46875be4a88

cc / @bewisesh91

Reply:
그리고 이쯤(16번째줄)에서 `showNumber.innerText = 0` 또는 `showNumber.innerText = resultNumber`같은 게 추가되어야 하지 않을까요? 그래야 result부분의 텍스트가 0으로 변경될 것 같습니다.

Reply:
오 그러면 0이 남네요. 근데 조금 헷갈리는게 showNumber의 경우 const로 했는데, innerText를 바꿔도 상관없는건가요?